requirements.txt

```diff
@@ -3,3 +3,7 @@ boto3 >= 1.7.13
 crc32c
 puremagic==1.4
 requests >= 2.18.4, < 3
+pyjwt >= 1.5.3, < 2
+cryptography >= 2.3, < 3
+furl >= 1.2.1
+hca
```

Comment:
removed `hca` dependency. It was add because because I saw it was used here https://github.com/HumanCellAtlas/dcplib/blob/dc2cfd154340d59cf35993f63288333f24ea5236/dcplib/etl/__init__.py#L23

Reply:
OK thanks - I will address that/add a comment there.